### PR TITLE
if else fscript

### PR DESCRIPTION
**Pull request by smirnovaka**

### CODE REVIEW

File: src/foam/parse/FScriptParserTest.js [hunk 1]
```diff
@@ -163,6 +163,15 @@ foam.CLASS({
     sps.setString("address.regionId.len==5");
     test(((Predicate) parser.parse(sps, px).value()).f(user), "address.regionId.len==10");
 
+    sps.setString("if ( address.regionId.len==5 ) { firstName } else { lastName.len+3 }");
+    test(((Expr) parser.parse(sps, px).value()).f(user) == "senorita", "if ( address.regionId.len==5 ) { firstName } else { lastName.len==3 ");
+
+    sps.setString("if ( address.regionId.len==4 ) { firstName } else { lastName.len+3 }");
+    test(((Double) ((Expr) parser.parse(sps, px).value()).f(user)) == 8, "if ( address.regionId.len==5 ) { firstName } else { lastName.len==3 ");
+
+    sps.setString("if ( address.regionId.len==4 ) { firstName } else { if ( lastName.len+3==10 ) { address.regionId } else { address.city } }");
```
Comment (smirnovaka) [inline src/foam/parse/FScriptParserTest.js:172]: @jlhughes examples of usage

Comment (kgrgreer) [reply src/foam/parse/FScriptParserTest.js:172]: nifty

File: src/foam/mlang/mlang.js [hunk 2]
```diff
@@ -4350,6 +4350,58 @@ foam.CLASS({
...
+
+  properties: [
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'predicate',
+      javaFactory: 'return foam.mlang.MLang.TRUE;'
+    },
+    {
+      class: 'foam.mlang.ExprProperty',
+      name: 'valueIfTrue'
```
Comment (kgrgreer) [inline src/foam/mlang/mlang.js:4366]: maybe better names: trueExpr and falseExpr

Comment (jlhughes) [reply src/foam/mlang/mlang.js:4366]: I've renamed these to trueExpr, falseExpr in my payblii branch.

**Pull request closed without merging**